CMakeLists.txt

```diff
@@ -80,3 +76,16 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
 	set_source_files_properties(${SG14_TEST_SOURCE_DIRECTORY}/plf_colony_test.cpp PROPERTIES
 		COMPILE_FLAGS "/wd4127") # Disable conditional expression is constant, use if constexpr
 endif()
+
+install(TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME}_targets)
+
+include(GNUInstallDirs)
+
+install(EXPORT ${PROJECT_NAME}_targets
+    NAMESPACE ${PROJECT_NAME}::
+    FILE ${PROJECT_NAME}-config.cmake
+    DESTINATION "${CMAKE_INSTALL_DATADIR}/cmake/${PROJECT_NAME}"
```

Comment:
Not sure whether this should be `CMAKE_INSTALL_DATADIR` or `CMAKE_INSTALL_LIBDIR`. Thoughts?

Reply:
If you search for config files in your system you will see both approaches. `GNUInstallDirs` documents that `LIBDIR` is supposed to be used for "object code libraries" while `DATADIR` should be used for "read-only architecture-independent data". It doesn't really matter since CMake will search in both places, but I think `DATADIR` is the more correct path.